src/Network/documentation/current-breaking-changes.md

```diff
@@ -48,7 +48,7 @@ The following cmdlets were affected this release:
 - The parameter `ProbeEnabled` is being removed
 
 **Add-AzureRmVirtualNetworkPeering**
-- The parameter alias `AlloowGatewayTransit` is being removed
+- The parameter alias `AllowGatewayTransit` is being removed
```

Comment:
@nschonni as mentioned previously, this can be reverted since this typo is correct